birdhouse/pavics-compose.sh

```diff
@@ -103,6 +104,7 @@ COMPOSE_DIR="`pwd`"
 # we source local configs, if present
 # we don't use usual .env filename, because docker-compose uses it
 [ -f env.local ] && . ./env.local
+. ./scripts/get-components-json.sh
```

Comment:
Given this is not a script that can be executed as all other script in folder `scripts/`, I think a more appropriate place for it would be right next to `pavics-compose.sh` with the name `get-components-json.inc.sh`.

`inc.sh` suffix to indicate it is meant to be included and not executed.  If you know other convention than `inc.sh` suffix to indicate it has to be sourced/included, please let me know.  I simply made that up !

Reply:
I'm not aware of any specific convention, but I think it is a good idea to have some "include" indication.

I prefer to leave it in `scripts/` though to avoid cluttering the `birdhouse` base directory. Otherwise, users will have a harder time figuring out which script is the entrypoint.